src/actions/index.js

```diff
@@ -35,24 +38,31 @@ export const setNumberOfLights = value => ({
   value
 });
 
-export const setActividadRecinto = value => ({
-  type: SET_ACTIVIDAD_RECINTO,
+export const setRoomActivity = value => ({
+  type: SET_ACTIVIDAD_ROOM,
```

Comment:
I think that this should be `SET_ACTIVITY_ROOM` , lmk what do you think about this @Lenninlasd .

Reply:
Done.